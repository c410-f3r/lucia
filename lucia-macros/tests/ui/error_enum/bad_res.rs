#[lucia_macros::pkg(api(Foo), data_format(json))]
mod pkg {
  #[derive(Debug)]
  #[pkg::req_data]
  pub struct Req;

  #[pkg::res_data]
  struct ResDSA;
}

fn main() {
}
