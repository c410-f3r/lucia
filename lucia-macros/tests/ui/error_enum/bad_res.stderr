error: Response data must end with the `Res` suffix.
 --> tests/ui/error_enum/bad_res.rs:8:10
  |
8 |   struct ResDSA;
  |          ^^^^^^
